edit-post/components/header/more-menu/index.js

```diff
@@ -25,19 +24,16 @@ const element = (
 			/>
 		) }
 		renderContent={ ( { onClose } ) => (
-			<div>
+			<div className="edit-post-more-menu__content">
 				<ModeSwitcher onSelect={ onClose } />
-				<div className="edit-post-ellipsis-menu__separator" />
 				<FixedToolbarToggle onToggle={ onClose } />
-				<div className="edit-post-ellipsis-menu__separator" />
-				<EditorActions />
+				<MenuItemsGroup
+					label={ __( 'Tools' ) }
+					filterName="editPost.MoreMenu.tools"
```

Comment:
Conceptually-speaking, why do we need two things - plugin slots and filterable menus - to express the same idea of "add to this menu" ? As of current master, we have both in this menu, and it's not clear why I would use one or the other, or why one can't be satisfied by the other.

Reply:
https://github.com/WordPress/gutenberg/blob/f175813c4871dd6472e33fd528bae79b6d9a24d0/edit-post/components/header/more-menu/index.js#L36-L40

Reply:
My guess is filterable menus were some of the earlier experiments, partly motivated by the emergence of `@wordpress/hooks`. They also felt cheaper (perhaps deceptively simple?). I'd say by now slots have established themselves as the standard extension method, though.

Reply:
cc @gziolo I think he was talking about removing those at some point?

Reply:
Yes, we should remove those filters and replace the only occurrence of hook based extension with slot and fill. As @mcsf already pointed out, this was an early exploration which turned out to be suboptimal from developer’s standpoint.

I had it planned to refactor on my todos list but it was never a priority. I will take care of it this week to avoid confusion.